### DISCUSSION TITLE
Snowflake vs Neon

### DISCUSSION THREAD

**Discussion by dalemccrory** [neondatabase/neon#1824] [Q&A] [answered]
I'm curious about your take on data architecture decisions made at the enterprise level and if Neon would help bridge some gaps. With a non-profit that doesn't have terabytes of data, outside consultants push the non-profit (us) to use Snowflake as the data warehouse (because it's cool and popular). However, our team uses postgres already and doesn't have huge amounts of money to spend on the solution. So we are caught between "the popular / expensive" option and the "common / you will have to migrate to Snowllake at some point" option. 

For data warehousing use cases, do you see Neon being a good postgres-compatible alternative to Snowflake? What would the limitations be?

**Comment by hlinnaka** [comment, marked as answer]
PostgreSQL works great for a data warehouse, if you don't have too much data. It has all the query features you need, it's just not as optimized for analytics queries as systems like Snowflake. So I'd suggest you give PostgreSQL a shot first and see how far it gets you.

Neon gives you simpler administration, ability to run read-only replicas etc. It doesn't have any special analytics features other than what's in PostgreSQL. However, it's serverless, which means you don't pay for it when you're not using it. That can be great for a data warehouse if you only need to run queries on it occasionally. Neon is currently in technical preview and the database size is limited to 10 GB. We will raise that limit - the architecture allows scaling up with no limits - but we're not there yet.
